ci-operator/step-registry/telcov10n/slcm/tests/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- natifridman
+- shaior
+- cplacani
+- ccardeno
```

Comment:
Actually, It is `ccardenosa` for GitHub account.

Reply:
done